draft-irtf-cfrg-vdaf.md

```diff
@@ -784,8 +784,8 @@ a particular encoding of that quantity as a byte string.
 Some common functionalities:
 
 * `additive_secret_share(x: list[F], num_shares: int, field: type[F]) ->
```

Comment:
`field: type[F]`
I consider this parameter is not needed, as it's an implementation detail.
This comment also applies to any other function with this parameter.